django/db/models/sql/compiler.py

```diff
@@ -1731,6 +1732,22 @@ def assemble_as_sql(self, fields, value_rows):
 
         return placeholder_rows, param_rows
 
+    def compile_where_clause_to_sql(self, expression):
```

Comment:
I'm thinking that this name feels a little bit generic...

Reply:
made a proposal for a name